WORKSPACE

```diff
@@ -1,260 +1,3 @@
-    integrity = "sha512-ULr0LDaEqQrMFGyQ3bhJkLsbtrQ8QibAseGZeaSUiT/6zb9IvIkomWHJIvgvwad+hinRAgsI51JcWk2yvwyL+w==",
-    package = "acorn",
-    # Root package where to link the package store
-    root_package = "",
-    version = "8.4.0",
-)
-
-############################################
-# Stardoc
-load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
-
-stardoc_repositories()
-
-load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
-
-rules_jvm_external_deps()
-
-load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
-
-rules_jvm_external_setup()
-
-load("@io_bazel_stardoc//:deps.bzl", "stardoc_external_deps")
-
-stardoc_external_deps()
-
-load("@bazel_features//:deps.bzl", "bazel_features_deps")
-
-bazel_features_deps()
-
-load("@stardoc_maven//:defs.bzl", stardoc_pinned_maven_install = "pinned_maven_install")
-
-stardoc_pinned_maven_install()
-
-# Buildifier
-load("@buildifier_prebuilt//:deps.bzl", "buildifier_prebuilt_deps")
-
-buildifier_prebuilt_deps()
-
-load("@buildifier_prebuilt//:defs.bzl", "buildifier_prebuilt_register_toolchains")
-
-buildifier_prebuilt_register_toolchains()
-
-# rules_lint
-load(
-    "@aspect_rules_lint//format:repositories.bzl",
-    "rules_lint_dependencies",
-)
-
-rules_lint_dependencies()
-
-load("@rules_multitool//multitool:multitool.bzl", "multitool")
-
-multitool(
-    name = "multitool",
-    lockfiles = [
-        "@aspect_rules_lint//format:multitool.lock.json",
-        "@aspect_rules_lint//lint:multitool.lock.json",
-    ],
-)
-
-load("@com_grail_bazel_toolchain//toolchain:deps.bzl", "bazel_toolchain_dependencies")
-
-bazel_toolchain_dependencies()
-
-load("@com_grail_bazel_toolchain//toolchain:rules.bzl", "llvm_toolchain")
-
-llvm_toolchain(
-    name = "llvm_toolchain",
-    llvm_version = "14.0.0",
-    sha256 = {
-        "darwin-aarch64": "1b8975db6b638b308c1ee437291f44cf8f67a2fb926eb2e6464efd180e843368",
-        "linux-x86_64": "564fcbd79c991e93fdf75f262fa7ac6553ec1dd04622f5d7db2a764c5dc7fac6",
-    },
-    strip_prefix = {
-        "darwin-aarch64": "clang+llvm-14.0.0-arm64-apple-darwin",
-        "linux-x86_64": "clang+llvm-14.0.0-x86_64-linux-gnu",
-    },
-    sysroot = {
-        "linux-aarch64": "@org_chromium_sysroot_linux_arm64//:sysroot",
-        "linux-x86_64": "@org_chromium_sysroot_linux_x86_64//:sysroot",
-        "darwin-aarch64": "@sysroot_darwin_universal//:sysroot",
-        "darwin-x86_64": "@sysroot_darwin_universal//:sysroot",
-    },
-    urls = {
-        "darwin-aarch64": ["https://github.com/aspect-forks/llvm-project/releases/download/aspect-release-14.0.0/clang+llvm-14.0.0-arm64-apple-darwin.tar.xz"],
-        "linux-x86_64": ["https://github.com/aspect-forks/llvm-project/releases/download/aspect-release-14.0.0/clang+llvm-14.0.0-x86_64-linux-gnu.tar.xz"],
-    },
-)
-
-load("@llvm_toolchain//:toolchains.bzl", "llvm_register_toolchains")
-
-llvm_register_toolchains()
+fail("Local WORKSPACE development unsupported")
```

Comment:
interesting, I haven't seen a `fail` in a WORKSPACE file before.

What does "local" mean here?
I think it's enough to just upgrade to Bazel 8 and not expect anyone will pass `--enable_workspace`

Reply:
Just means you'll see that `fail` message instead of random bazel errors saying things our undefined.

I guess I meant "running bazel in this directory". You can still use rules_js from a different workspace, it's just development within the `rules_js` repository that won't work...